components/edit_channel_header_modal/edit_channel_header_modal.jsx

```diff
@@ -25,43 +21,49 @@ const holders = defineMessages({
     }
 });
 
-class EditChannelHeaderModal extends React.Component {
+class EditChannelHeaderModal extends React.PureComponent {
+    static propTypes = {
+        intl: intlShape.isRequired,
+        onHide: PropTypes.func.isRequired,
+        channel: PropTypes.object.isRequired,
+        ctrlSend: PropTypes.bool.isRequired,
+        serverError: PropTypes.object,
+        requestStatus: PropTypes.string.isRequired,
+        onPatchChannel: PropTypes.func.isRequired
+    }
+
     constructor(props) {
         super(props);
 
         this.handleChange = this.handleChange.bind(this);
         this.handleSave = this.handleSave.bind(this);
         this.handleKeyDown = this.handleKeyDown.bind(this);
         this.handleKeyPress = this.handleKeyPress.bind(this);
-        this.onShow = this.onShow.bind(this);
         this.onHide = this.onHide.bind(this);
-        this.handlePostError = this.handlePostError.bind(this);
         this.focusTextbox = this.focusTextbox.bind(this);
```

Comment:
Should remove all function binding. (Prefer `fn = () => {}`, see requested change below.)